src/btree/bt_vrfy_dsk.c

```diff
@@ -293,7 +293,7 @@ __verify_dsk_row(
 		case WT_CELL_ADDR_LEAF_NO:
 		case WT_CELL_KEY_OVFL:
 		case WT_CELL_VALUE_OVFL:
-			if (!bm->addr_valid(bm,
+			if (bm->addr_invalid(bm,
 			    session, unpack->data, unpack->size))
```

Comment:
With that change, the only error return from `WT_BM::addr_invalid` is `EINVAL`.

I mostly agree about checking for specific returns. Obviously, the problem is a future change to `__wt_vunpack_uint` can now break the verify code, and it's impossible to check every possible consumer every time an underlying function changes its set of error values. I agree in this case that's pretty unlikely -- the trick is drawing the line between places it's OK to do this, and places it's not.